Tribler/Test/Community/Multichain/statistics/test_page_rank.py

```diff
@@ -0,0 +1,23 @@
+from unittest import TestCase
+
+from networkx import gnm_random_graph
+
+from Tribler.community.multichain.statistics.page_rank import IncrementalPageRank
+
+
+class TestIncrementalPageRank(TestCase):
+    """
+    As it is a Monte Carlo method, integration tests can only be basic.
+    """
+    def setUp(self):
+        nr_of_nodes = 1000
```

Comment:
This look like some magic numbers to me. Think about a config for the test cases.